tox.ini

```diff
@@ -64,6 +64,7 @@ setenv =
 	DJANGO_SETTINGS_MODULE = test_settings
 deps = 
 	-r{toxinidir}/requirements/test.txt
-commands = 
+	Django>=2.2,<2.3
```

Comment:
Why this special cal out? Instead of adding django to the test.in file or something?

Reply:
We only want to run `pii_check` test against `Django22`. Adding the django constraint in the `test.txt` file will cause all the tests to fail against other django versions so we added the condition separately.